campaignSpec.md

```diff
@@ -76,6 +77,21 @@ Rules that apply to submitting events
 
 `{ allow: [{ uids: [channel.creator] }, { uids: null, rateLimit: { type: "ip", timeframe: 1000 } }] }` - this will allow the creator to submit as many events as they like, but everyone else will be restricted to 1 event per second per IP
 
+#### PriceMultiplicationRules
+
+The `multiplier/amount` is mandatory and `multiplier` is a float, `amount` is a BigNumber, all the others are optional and are arrays of possible values to match. 
+
+* `multiplier / amount`: multiplier or amount in float & BigNumber format respectively
+* `evType`: an array of event types e.g. `['IMPRESSION']`
```

Comment:
all of the arrays need to be optional, where not providing it means "match everything"; e.g. not providing a `publisher` (or `publisher: null`) means "match any publisher"